sdk/eventhubs/test-resources.json

```diff
@@ -8,10 +8,6 @@
                 "description": "Base name to append to Azure resources."
             }
         },
-        "storageEndpointSuffix": {
```

Comment:
Before merging this in and removing the Storage account creation for EventHubs could you double check that Checkpoint Store doesn't use it at all. I know it is built to use Storage Blobs to store checkpointing data, if there are no integration tests than this can be completely removed.

Reply:
Yes. We do not have any integration tests for Checkpoint store. It seems wasteful to create something that is unused at the moment. If we add some in, we can do a `git fl` and resurface the code.